docs/src/examples/components/Popup/Types/PopupExample.shorthand.tsx

```diff
@@ -1,6 +1,25 @@
-import * as React from 'react'
-import { Button, Popup } from '@stardust-ui/react'
+import React from 'react'
+import { Button, Popup, Dialog } from '@stardust-ui/react'
 
-const PopupExample = () => <Popup trigger={<Button icon="more" />} content="Hello from popup!" />
+const PopupExample = () => (
+  <>
```

Comment:
Disregard this example for now, will add separate for all components which are using the PortalInner